.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -39,20 +41,23 @@ Please see the text preview to avoid unnecessary formatting errors.
 2. ...
 3. ...
 
-### Logs
 
-<!--
-Logs are always required for a bug report, use `-l debug` [(help)](https://streamlink.github.io/cli.html#cmdoption-l) and avoid using `--stream-url` as that will suppress log output.
+### Log output
 
-Make sure to **remove username and password**
+<!--
+TEXT LOG OUTPUT IS REQUIRED for a bug report!
+Use the `--loglevel debug` parameter and avoid using parameters which suppress log output.
```

Comment:
Should we add an example of what the streamlink command looks like when running with `-l debug` here? Some people have been confused about this lately.

Reply:
I think that would make it a bit too long